docs/reference/workflows/configuring_jobs.rst

```diff
@@ -64,7 +64,19 @@ For example, if a job is defined as follows:
     EXECUTABLE script.sh
     STOP_ON_FAIL TRUE                   -- Tell the job to stop ert on failure
 
-STOP_ON_FAIL can also be specified within the internal (python) or external (executable) job script.
+In case of an external sh job script, you have to manually set the stop on fail option (-e)
+
+::
+    set -e
+
+For example, this bash external job script will stop on failure
+
+::
+    #/usr/bin/env bash
```

Comment:
Hm, shouldn't it be: `#!/bin/bash` only?

Reply:
It can be both, but I will change it anyways.